ark/include/kernels/common/unit_op.h

```diff
@@ -51,14 +51,14 @@ struct UnitOp {
     static_assert(_UnitOutDims::W > 0,
                   "Unit dimension is not positive in dimension W");
 
-    static_assert(_OutDims::N % _UnitOutDims::N == 0,
-                  "Dimension N is not divisible by the unit dimension");
-    static_assert(_OutDims::C % _UnitOutDims::C == 0,
-                  "Dimension C is not divisible by the unit dimension");
-    static_assert(_OutDims::H % _UnitOutDims::H == 0,
-                  "Dimension H is not divisible by the unit dimension");
-    static_assert(_OutDims::W % _UnitOutDims::W == 0,
-                  "Dimension W is not divisible by the unit dimension");
+    // static_assert(_OutDims::N % _UnitOutDims::N == 0,
```

Comment:
No needed?

Reply:
Updated in [ecd5afd](https://github.com/microsoft/ark/pull/205/commits/ecd5afd22a3626f27ad476e2094d00f294492d89).